.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
 * @suzil
+* @erwanp
+* @arunavabasu-03 
```

Comment:
Because this deploys to my personal AWS account, I'd like to have a final check on whatever gets merged (though I am more than open to moving hosting of this off of my own AWS account). This CODEOWNERS file sets it up that I have to approve before stuff gets merged.